app/views/posts/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1>List of all posts from the given user id</h1>
\ No newline at end of file
+<div class='template'>
+  <%= render partial: 'shared/user-container', locals: { user: @user } %>
+  <% @posts.each do |post| %>
+    <div class='post-container'>
+      <div>
+        <h3><%= link_to post.title, user_post_url(@user, post) %></h3>
```

Comment:
- [ ] also the profile picture is not showing, kindly fix it as to fit the wireframe